lib/mobility-core/src/Kernel/External/Payment/Interface/Juspay.hs

```diff
@@ -238,7 +238,9 @@ mkOrderStatusResp Juspay.OrderData {..} =
           mandateFrequency = justMandate.frequency,
           mandateMaxAmount = justMandate.max_amount,
           payerVpa = payer_vpa,
-          upi = castUpi <$> upi
+          upi = castUpi <$> upi,
+          refunds = maybe [] mkRefundsData refunds,
+          amountRefunded = realToFrac <$> amount_refunded
```

Comment:
Does this mean amount is already refunded? Or are we storing the amount to be refunded here?

Reply:
@NikithShetty it means the amount refunded so far
and refunds has details of all the refunds request called
<img width="563" alt="Screenshot 2024-05-20 at 8 36 56 PM" src="https://github.com/nammayatri/shared-kernel/assets/59287619/ac6b61ec-c980-4d5a-b57c-f03132bd5949">